tests/Functional/DependencyInjection/ServiceTest.php

```diff
@@ -45,3 +63,17 @@ public function testCanBeLoaded()
         }
     }
 }
+
+class ServicesPublicPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        foreach ($container->getServiceIds() as $id) {
+            if (strncmp('fos_http_cache.', $id, 15)) {
+                continue;
+            }
+
+            $container->setDefinition($id, $container->getDefinition($id)->setPublic(true));
```

Comment:
Why does it set the same definition with the same id again? Just `$container->getDefinition($id)->setPublic(true)` should be enough.